quinn-h3/src/server.rs

```diff
@@ -8,7 +8,7 @@ use std::{
 
 use futures::{ready, Stream};
 use http::{response, Request, Response};
-use quinn::{CertificateChain, EndpointBuilder, PrivateKey, RecvStream, SendStream, self};
+use quinn::{self, CertificateChain, EndpointBuilder, PrivateKey, RecvStream, SendStream};
```

Comment:
This really belongs in the previous commit.

Reply:
rebased in previous commit.